_posts/2016-06-20-foreman-1.11-dhcp-bugs.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: post
+title: Foreman 1.11.2 DGCP bugs
```

Comment:
Typo? DHCP right?

Could you make this clearer about the focus of the content? Perhaps "Working around the Foreman 1.11.2 DHCP bugs"?